inception-project/inception
# #1162 - Update checkstyle maven plugin

**What's in the PR**
- Update plugin
- Update checkstyle version
- Add workaround for Eclipse checkstyle integration quirk

**How to test manually**
* Import project into an Eclipse which is prepped with the Checkstyle Eclipse integration plugins and see that the settings from Maven are properly recognized
* Run a build and see that checkstyle doesn't crash

**Automatic testing**

**Documentation**
